### ISSUE TITLE
FE|  Profile | Accessibility | Screenreader on DD FAQs

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#47003]
## Background

When editing Direct deposit/comp & pension benefits, the initial focus is placed on the first editable field which is the routing info. If you arrow up and open the FAQ, JAWS will not see the paragraph detailing how to get the routing info.

The proposed solution by the a11y team is to:

> Set focus to the h2
> 
> The paragraph with the information is a figure caption. Removing it from the figure element and placing it in a plain paragraph below the image will solve the problem.
> 

Additional background from original ticket is here: #46614.

## Tasks
- [x] Update the modal to follow new [mock](https://www.sketch.com/s/1a920e73-1dcb-47c4-aae8-08656756c131/a/4a38jpJ)
- [x] Conduct Unit tests 
- [x] Run E2E tests covering current code and regression
- [x] All axe checks pass

## Acceptance Criteria
- [x] Complete all unit testing
- [x] End-to-end tests showing 0 violations
- [x] Notify Product Manager that the change is ready to be validated on Staging

**Comment by SKasimow** [comment]
Addresses #47003

**Comment by SKasimow** [comment]
@adamwhitlock1 - so we need someone from a11y team to validate?

**Comment by SKasimow** [comment]
@TiffanyPender - would you be able to validate this for us?

**Comment by SKasimow** [comment]
Hi @TiffanyPender - are you able to validate this work for us?

**Comment by SKasimow** [comment]
Hi @davidakennedy - could we have an accessibility person validate the fix for this ticket?

**Comment by TiffanyPender** [comment]
I can take a look at this @SKasimow. Apologies for missing this.

**Comment by SKasimow** [comment]
@TiffanyPender no worries; thank you for helping us with this

**Comment by TiffanyPender** [comment]
I've checked this on staging @SKasimow. It seems that the focus still goes to the routing number field after clicking the EDIT button on the Direct Deposit tab instead of the suggested H2. I also checked that JAWS now reads the informational text below the check image when manually navigating backwards.

**Comment by SKasimow** [comment]
@adamwhitlock1 - can you take a look and advise?

**Comment by adamwhitlock1** [comment]
@SKasimow I have pushed out a PR that will resolved the focus when edit is initiated.

@TiffanyPender When you have a moment, would you take a look on staging for direct deposit and let me know if the new code accomplishes the focus management appropriately. Thanks!
